include/simdjson/portability.h

```diff
@@ -15,6 +15,43 @@
 #define IS_ARM64 1
 #endif
 
+#if defined(__clang__) && defined(_MSC_VER) && defined( IS_X86_64)
+// clang under Visual Studio needs help
+#define __AVX2__ 1
+#define __AVX__ 1
+#define __SSE3__ 1
+#define __SSE4_1__ 1
+#define __SSE4_2__ 1
+#define __SSE_MATH__ 1
+#define __SSE__ 1
+#define __SSSE3__ 1
+#define __PCLMUL__ 1
+#define __BMI2__ 1
+#define __BMI__ 1
+#define __LZCNT__ 1
+// clang under Windows has _stricmp (like visual studio) but not strcasecmp (as clang normally has)
+#ifndef strcasecmp
+#define strcasecmp _stricmp
+#endif
+// clang under windows has _aligned_malloc but not posix_memalign as clang normally does
+static inline int posix_memalign(void **p, size_t alignment, size_t size) {
+#ifdef _MSC_VER
```

Comment:
I assume this was copied from somewhere else, but in this context _MSC_VER == 1

Reply:
I thought that _MSC_VER was defined to be the visual studio version?

Reply:
Sorry, I just mean this will always evaluate to true because you are already in an if _MSC_VER block :)

Reply:
It should be #undef. I wonder how I made this typo. Thanks.

Reply:
Oh. No. It is correct.

Right, there is code duplication because the taken path is obvious. It is indeed copied from elsewhere but the goal here would to generalize this over time to support other windows-based systems.

Anyhow, this PR is not going to get merged as such.